docs/messaging.rst

```diff
@@ -170,7 +174,7 @@ Every message is serialized to a sequence of at least six blobs of bytes:
       b'{parent_header}', # serialized parent header dict
       b'{metadata}',      # serialized metadata dict
       b'{content}',       # serialized content dict
-      b'blob',            # extra raw data buffer(s)
+      b'0x1234',          # extra raw data buffer(s)
```

Comment:
The `0x` here suggests that it is always hex-encoded, which as far as I know is not the case.